src/efs_parser/BigBed.py

```diff
@@ -105,3 +105,18 @@ def parseLeafDataNode(self, chrmId, start, end, zoomlvl, rStartChromIx, rStartBa
                             x += 1
                     x += 2
         return result
+    def groupRows(self,res):
+        result = pd.DataFrame(columns=res.columns)
```

Comment:
res should already be a dataframe at this point, so we don't need to reconvert this again

Reply:
I am creating an empty dataframe (result) that takes the structure of (res).
and the result will build up as we go through the loop